packages/valory/skills/abstract_round_abci/abci_app_chain.py

```diff
@@ -31,6 +32,8 @@
 )
 
 
+logger = logging.getLogger(__name__)
```

Comment:
recommended logger usage for consistency: https://github.com/valory-xyz/open-autonomy/blob/main/packages/valory/connections/http_client/connection.py#L55